### DISCUSSION TITLE
Personalizar barra de desplazamiento fácilmente

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#5] [Announcements]
# Personalizar barra de desplazamiento fácilmente

La barra de navegación es un elemento común en cualquier aplicación y en este
  artículo te ayudaré a modificar su aspecto

https://www.zkreations.com/2018/02/personalizar-scrollbar-facil-custom-scrollbars.html

**Comment by danieIabel** [comment]
Ahora los navegadores basados en Chromium soportan las propiedades `scrollbar-color` y `scrollbar-width`, por lo cual el scroll no se apreciaba adecuadamente en estos navegadores. He solucionado el problema agregando una condición al codigo.
